Archives/VolFile.cpp

```diff
@@ -132,7 +129,7 @@ namespace Archives
 			
 			std::vector<void*> packedFileBuffer;
```

Comment:
This data type still needs fixing. Should by a byte sized data type. Pointers are not byte sized. When the buffer is resized, it allocates `sectionHeader.length * sizeof(void*)` bytes. For a 32-bit build, pointers are 4 bytes, so that is 4x more memory than needed, and for a 64-bit build, that is 8x more memory than needed. Plus the type just doesn't make sense in this context.

The vector should contain a hidden internal pointer to an array of bytes, not a pointer to an array of pointers to unknown type. You have too much indirection here. :wink:

Reply:
Thanks for re-pointing this out. I was going to research what size of void* was after you last pointer. I'll plan to update to uint8_t as this makes more since to me than char since it isn't supposed to be text.

Reply:
Not sure if it was intended, but I appreciate your puns. :smiley: